### DISCUSSION TITLE
why is the documentation is soo poor?

### DISCUSSION THREAD

**Discussion by 7odaifa-ab** [lepoco/wpfui#1178] [General]
all the documentation is poorly maintained, the gallery app shows the wrong XMAL, and I found nothing regarding the system tray and how to add them!

**Comment by pomianowski** [comment]
Hey, thanks for your interest in WPF UI. It's an open source project mainly maintained by one person without financing. If you want to improve the documentation, you are welcome to edit the docs catalog in the master branch.

**Reply by 7odaifa-ab** [reply]
sure yeah if I could understand the flow of it, but working on such a huge and outstanding project that can only be used by 10% of people is just vain

**Reply by DaveFace** [reply]
The people looking for documentation aren't in a position to write it, though: documentation has to start with people who understand what they're documenting. I really wish I could help but I'm not even sure my high level undersetanding of WPF UI is accurate.

**Reply by Veritera** [reply]
"mainly maintained by one person" so your 69 contributors doesn't exist?

**Comment by Apskalle** [comment]
Leszek is doing a wonderful job in keeping wpf up to date, something even MS couldnt muster.   
So comments like that are not very helpful - try to contribute instead.

**Reply by 7odaifa-ab** [reply]
"The people looking for documentation aren't in a position to write it" 
you need to know the importance of documenting, it's more important than keeping it up to date!
I know how much hard work is put into this, but it's useless if it's not able to be used, I'm not a hater I do, in fact, I use this library [https://github.com/7odaifa-ab/DDNS-Cloudflare-API](https://github.com/7odaifa-ab/DDNS-Cloudflare-API) but as you can see I'm always constraint of what I can do with it I cannot pass what functions the initial template provided

**Reply by Xenovore** [reply]
> "The people looking for documentation aren't in a position to write it"

100% this. 

Some of us are just starting to use wpfui. We need quality documentation to even understand how stuff is supposed to work! And so far, in my experience, the docs seem incomplete and sometimes are just wrong.

So: How can anyone contribute anything meaningful if they are struggling to understand how to use it in the first place?

**Comment by Apskalle** [comment]
Hi there,

WPF UI is 97% standard WPF so if you are struggling with the concepts you likely need to read up on wpf basics.
Remaining 3% could be somewhat new things but even here you should be able to make it by experimenting a bit.

Just my 5¢,

/M

**Reply by 7odaifa-ab** [reply]
i don't think you have used this library lately...
